sentry-android-core/build.gradle.kts

```diff
@@ -79,6 +79,7 @@ dependencies {
 
     // libs
     implementation(Config.Libs.gson)
+    implementation("com.android.support:support-annotations:28.0.0")
```

Comment:
Seems required to implement `FileObserver`.
If that's it, I'll just add it to `Config.Libs.support-annotations`

Reply:
I'd not say this is required, but if so, prefer the androidx annotations
https://android.googlesource.com/platform/frameworks/support.git/+/refs/heads/androidx-annotation-release/annotation/annotation/src/main/java/androidx/annotation

Reply:
The annotation is part of the signature and if I deleted it, it wouldn't compile.
I can try getting rid of it again but I recall not building otherwise

Reply:
`android.support` is deprecated and because of that, they made a tool called `jetifier` to migrate `android.support` . to `androidx` libs., but `androidx` is dependent of the framework so would be better to use `jetbrains-annotations` to be able to reuse/understand on the pure java layer.
ref https://developer.android.com/studio/command-line/jetifier